### PR TITLE
When multiple files are selected, open them together

### LINKED ISSUE #7530: The songs will not automatically enqueue by selecting multiple music files and play
### Description

When highlighting multiple music files and pressing enter to play, only one music file will be played and not enqueue all the music files selected.

### Steps To Reproduce

1. Go to the folder containing music files.
2. Highlight multiple music files.
3. Press enter to play the selected music files.
4. Only one music file will be played, other selected music files will not be enqueued.

### Expected behavior

All the selected music files should be enqueued and played in a list, but not only one music file will be played.

### Files Version

Version: 2.0.34.0 OS Version: 10.0.22000.376

### Windows Version

Windows 11 21H2 22000.376

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/7844441/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/7844442/debug_fulltrust.log)

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #7530

**Details of Changes**
Add details of changes here.
- If multiple files are selected, use FTP to open them together so e.g all selected music files will play in the default app.

### CODE REVIEW

File: src/Files/Helpers/NavigationHelpers.cs [hunk 1]
```diff
@@ -83,17 +83,29 @@ public static async void OpenSelectedItems(IShellPage associatedInstance, bool o
...
-                var type = item.PrimaryItemAttribute == StorageItemTypes.Folder ?
-                    FilesystemItemType.Directory : FilesystemItemType.File;
+                // Multiple files are selected, open them together
+                opened = await Win32Helpers.InvokeWin32ComponentAsync(string.Join('|', selectedItems.Select(x => x.ItemPath)), associatedInstance);
+            }
+            if (!opened)
+            {
+                foreach (ListedItem item in selectedItems)
+                {
+                    var type = item.PrimaryItemAttribute == StorageItemTypes.Folder ?
```
Comment (d2dyno1) [inline src/Files/Helpers/NavigationHelpers.cs:100]: `var isFolder` ?

File: src/Files/Helpers/Win32Helpers.cs [hunk 2]
```diff
@@ -45,7 +45,10 @@ public static async Task InvokeWin32ComponentsAsync(IEnumerable<string> applicat
                 }
 
                 await connection.SendMessageAsync(value);
+                return true;
```
Comment (d2dyno1) [inline src/Files/Helpers/Win32Helpers.cs:48]: How about also getting the response, and if it's "Success", then return `true`?

Comment (gave92) [reply src/Files/Helpers/Win32Helpers.cs:48]: Currently the ftp does not return any response for this request, but it's a good idea to change that.

Comment (d2dyno1) [reply src/Files/Helpers/Win32Helpers.cs:48]: Either assume the operation completed successfully if it doesn't report the status or like you've said, update the ftp to also report "Success" 👍

Comment (gave92) [reply src/Files/Helpers/Win32Helpers.cs]: Forgot SendMessageAsync() actually returns a result, changed function to return that.

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**